### PR TITLE
Retrieve expected data from all file locations.

### PR DESCRIPTION
**Pull request by whimboo**
Fixes issue #10.

### CODE REVIEW

**Comment by whimboo** [comment]
@mathiasbynens can you please review and merge? Thanks!

File: preprocess-data.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 const fs = require('fs').promises;
 const fetch = require('node-fetch');
 
+const baseURL = "https://hg.mozilla.org/mozilla-central/";
+
+const filePaths = [
+  "remote/test/puppeteer-expected.json",
+  "remote/puppeteer-expected.json",
+];
+
```
Comment (mathiasbynens) [inline preprocess-data.js:10]: style nit for consistency: 

```suggestion
const baseURL = 'https://hg.mozilla.org/mozilla-central/';

const filePaths = [
  'remote/test/puppeteer-expected.json',
  'remote/puppeteer-expected.json',
];

```

**Review by mathiasbynens — APPROVED** [review]

**Comment by whimboo** [comment]
Thanks!

**Comment by whimboo** [comment]
@mathiasbynens how long will it actually take until the page will reflect the new data?

**Comment by whimboo** [comment]
Oh wait. We would have to push to 'gh-pages'. I haven't done that for a while. Could you please take care of it?

**Comment by mathiasbynens** [comment]
That's supposed to happen automatically via GitHub Actions, but apparently CI disabled itself because there had been no repository activity in the last 60 days (I didn’t know that was a requirement!). I’ve just re-enabled it, and made it possible to force manual CI runs here: 637eae080008b076eb2f7da7b2b491aca149bbb2

**Pull request merged**